File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.cs
using System;

public class Solution 
{
    public int solution(int[] num_list) 
    {
        int numMul = 1;
        int nunSum = 0;
        int answer = 0;
        
        for(int i=0; i<num_list.Length; i++){
            numMul *= num_list[i];
            nunSum += num_list[i];
        }
        
        return answer = (num_list.Length >= 11) ? nunSum : numMul;
    }
}

File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list)
{
    int size = num_list.size();
    int ans;
    if(size >= 11)
    {
        ans = 0;
        for(int n : num_list)
        {
           ans += n; 
        }
    }
    else
    {
        ans = 1;
        for(int n : num_list)
        {
           ans *= n; 
        }
    }
    return ans;
}